makedist

```diff
@@ -40,8 +40,8 @@ if test "${2}" -lt "13" -o "${2}" -eq "13" -a "${3}" -lt "5"; then
 fi
 IFS="$old_IFS"
 
-if test "x$PHPROOT" = "x"; then
-    PHPROOT=git@git.php.net:php-src.git;
+if test "x$PHPROOT" -ne "x"; then
```

Comment:
from man test

```
       STRING1 != STRING2
              the strings are not equal
       ...

       INTEGER1 -ne INTEGER2
              INTEGER1 is not equal to INTEGER2
```

So you have to use != instead of -ne

Reply:
Thank you!